src/views/download/download.scss

```diff
@@ -6,123 +6,171 @@
 }
 
 .download {
-    .title-banner {
-        &.masthead {
-            margin-bottom: 0;
-            background-color: $ui-blue-dark;
-            padding-bottom: 0;
+    .title {
+        margin-bottom: 1rem;
+        font-size: 2rem;
+    }
+
+    .centered {
+        display: flex;
+    }
+
+    .self-center {
```

Comment:
instead of naming this class after what it does, I suggest naming it after the type of component/tag that uses it. In this case, you might call it 'icon'?

Reply:
Done! Actually, there already was an icon for exactly this... so I just moved it there and removed this class altogether :)